packages/lib/src/vis/shared/LineSelectionMesh.tsx

```diff
@@ -1,14 +1,8 @@
-import type { MeshProps } from '@react-three/fiber';
-import type { Vector2 } from 'three';
-
 import SelectionLine from './SelectionLine';
+import type { SelectionMeshProps } from './SelectionMesh';
 import SelectionMesh from './SelectionMesh';
 
-interface Props extends MeshProps {
-  onSelection?: (startPoint: Vector2, endPoint: Vector2) => void;
-}
-
-function LineSelectionMesh(props: Props) {
+function LineSelectionMesh(props: SelectionMeshProps) {
```

Comment:
`modifierKey` is added in `SelectionMeshProps`. I added this interface to avoid copying the `Props` between this mesh and `RectSelectionMesh`